python/paddle/fluid/incubate/fleet/parameter_server/pslib/optimizer_factory.py

```diff
@@ -328,13 +328,10 @@ def _check_config_fleet_with_program_op(self, strategy, table_name,
             strategy[table_name] = dict()
         st = strategy[table_name]
 
-        accessor = None
+        accessor = "DownpourCtrAccessor"
```

Comment:
why DownpourCtrAccessor default

Reply:
If use doesn't config sparse_accessor_class in config_fleet.py，we set "DownpourCtrAccessor" as default accessor